### ISSUE TITLE
onScanResults(scanResults) never fires on Edge 1030 plus

### ISSUE THREAD

**Issue by cyberman54** [markdotai/emtb#6]
I built the software and sideloaded it on my edge 1030 plus. It turned out,` onScanResults() `never fires after a scan has succesfully started (reporting state=1/status=0 in log). I am facing this problem with another BLE app, too. Thus, i assume it may depend on my toolchain, which is Windows10/VS Code/SDK 4.1.4. Can you help here?

**Comment by markdotai** [comment]
It sounds like it could match this firmware bug:  https://forums.garmin.com/developer/connect-iq/i/bug-reports/bluetooth-ciq-datafield-break-in-fw-9-1

They updated it recently to say they have a fix, but it will be in a future firmware release.

Does it sound the same as your problem too?

**Comment by cyberman54** [comment]
Indeed, that bug described exactly my problem with edge1030+. Will try on a 1030no+ and will try downgrade the 1030+. Thanks for the hint.
